MediaGalleryUi/Test/Unit/Model/ConfigTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+namespace Magento\MediaGalleryUi\Test\Unit\Model;
+
+use Magento\MediaGalleryUi\Model\Config;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * Config data test.
+ */
+class ConfigTest extends TestCase
+{
+    private const XML_PATH_ENABLED = 'media/gallery/enabled';
+
+    /**
+     * @var ObjectManager
+     */
+    private $objectManager;
+
+    /**
+     * @var Config
+     */
+    private $config;
+
+    /**
+     * @var ScopeConfigInterface|MockObject
+     */
+    private $scopeConfigMock;
+
+    /**
+     * Prepare test objects.
+     */
+    protected function setUp(): void
+    {
+        $this->objectManager = new ObjectManager($this);
+        $this->scopeConfigMock = $this->createMock(ScopeConfigInterface::class);
+        $this->config = $this->objectManager->getObject(
+            Config::class,
+            [
+                'scopeConfig' => $this->scopeConfigMock
+            ]
+        );
+    }
+
+    /**
+     * Get target environment test.
+     */
+    public function testIsEnabled(): void
+    {
+        $this->scopeConfigMock->expects($this->once())
+            ->method('isSetFlag')
+            ->with(self::XML_PATH_ENABLED)
+            ->willReturn(true);
+        $this->config->isEnabled();
```

Comment:
This test does not actually test the result of `isEnabled` method call. I'd use a data provider to check both `true` and `false` values and add an assertion:
```suggestion
        $this->assertEquals(true, $this->config->isEnabled());
```